### PR TITLE
Add integration test for tiup-dm

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
- Add integration test for tiup-dm
- Remove dm-portal since it's deprecated in the plan by DM team

### What is changed and how it works?
add `test_cmd` and `test_upgrade` like tiup-cluster to test commands of tiup-dm

### CODE REVIEW

File: templates/scripts/run_dm-master_scale.sh.tpl [hunk 1]
```diff
@@ -17,9 +17,9 @@ cd "${DEPLOY_DIR}" || exit 1
 {{- end}}
 
 {{- if .NumaNode}}
-exec numactl --cpunodebind={{.NumaNode}} --membind={{.NumaNode}} bin/bin/dm-master \
+exec numactl --cpunodebind={{.NumaNode}} --membind={{.NumaNode}} bin/dm-master/dm-master \
```
Comment (lonng) [inline templates/scripts/run_dm-master_scale.sh.tpl:20]: The directory layout seems weird, why not `bin/dm/dm-master` or `bin/dm/dm-worker`? Or place the `dm-master/worker` in `bin` directly?

File: tests/tiup-dm/test_upgrade.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```
Comment (lucklove) [inline tests/tiup-dm/test_upgrade.sh:1]: Please test with`--native-ssh` flag too.

File: templates/scripts/run_dm-master_scale.sh.tpl (hunk 1 above)
Comment (july2993) [reply templates/scripts/run_dm-master_scale.sh.tpl:20]: same issue as https://github.com/pingcap/tiup/pull/554#discussion_r448338005
I think it need some refactor to it, or it's restricted by the tar bar format.

there are existing such weird in cluster:
```
➜  tiup git:(dm_test) ✗ ag bin/bin ./templates
templates/scripts/run_grafana.sh.tpl
25:exec numactl --cpunodebind={{.NumaNode}} --membind={{.NumaNode}} bin/bin/grafana-server \
27:exec bin/bin/grafana-server \
```

Comment (lonng) [reply templates/scripts/run_dm-master_scale.sh.tpl:20]: I think the tarball layout should be changed to make the final layout more natural.

File: tests/tiup-dm/test_upgrade.sh (hunk 2 above)
Comment (july2993) [reply tests/tiup-dm/test_upgrade.sh:1]: https://github.com/pingcap/tiup/pull/629/commits/e809ad9a0f03a80039986743f6f2911b28df0f3d

**Comment by july2993** [comment]
@lucklove PTAL

**Pull request merged**